config/sidebar.yml

```diff
@@ -208,8 +208,12 @@ articles:
       - title: "Social/Enterprise"
         url: "/identityproviders"
         children:
+        
           - title: "Custom Social Extensions"
             url: "/extensions/custom-social-extensions"
+            
+          - title: "Passing Parameters to IdPs"
+            url: "/connections/passing-parameters-to-idps
```

Comment:
missing double quote at the end of the line
`"/connections/passing-parameters-to-idps` -> `"/connections/passing-parameters-to-idps"`

(causes build to fail)